src/main/java/com/google/sps/data/Comment.java

```diff
@@ -33,10 +33,10 @@ public boolean equals(Object o) {
             && that.getDisplayName().equals(this.name);
     } 
 
-    @Override
-    public int hashCode(){
-        return 42;
-        //TODO(swetha-gangu): implement hashCode()
+    @Override 
+    public int hashCode() {
+        int prime = 31;
```

Comment:
prefer using Objects.hash(name, comment) instead of rolling your own algorithm.

https://docs.oracle.com/javase/7/docs/api/java/util/Objects.html#hash(java.lang.Object...)